config/locales/translations/zendesk_apps_support.yml

```diff
@@ -100,13 +108,17 @@ parts:
       title: "App builder job: missing translation file"
       value: "Missing translation file for locale '%{defaultLocale}'. Please read: http://developer.zendesk.com/documentation/apps/i18n.html"
   - translation:
+      key: "txt.apps.admin.error.app_build.invalid_host"
+      title: "App builder job: invalid host application for apps"
+      value: "%{host_name} is an invalid host for apps."
+  - translation:
       key: "txt.apps.admin.error.app_build.invalid_location.one"
       title: "App builder job: invalid locations"
-      value: "%{invalid_locations} is an invalid location."
+      value: "%{invalid_locations} is an invalid location in %{host_name}."
```

Comment:
Please provide the example values of `%{invalid_locations}` & `%{host_name}` for translators in `title`. Thanks!